salt/apps/ocw/templates/webhook-publish.sh.jinja

```diff
@@ -14,6 +11,16 @@ lock_dir=/tmp/webhook-publish-lock
 # that came in during the period of the current run.
 retry_file=/tmp/webhook-publish-retry
 
+# Configuration for hugo-course-publisher
+OCW_TO_HUGO_INPUT=/opt/ocw/open-learning-course-data
+OCW_TO_HUGO_DOWNLOAD=1
```

Comment:
You don't need to set this flag I think, as `ocw-to-hugo` is not handling the course download and you are running `aws sync` to sync `open-learning-course-data-production` right?

Reply:
Although this made me think, since we're going to need to support updating only a single / a few courses in the future, we could use this functionality to accomplish that.  I'll need to add support for not blowing away the built site every time a build runs, but in theory we could use the download functionality already built into `ocw-to-hugo` to just only download a subset of updated courses, process them into markdown and build replacement course sites.

Reply:
I removed that `aws sync`, which used to be on lines 54 through 58, in order to simplify things. It's nice that that could make publishes of subsets feasible in the future.

Reply:
Ah, I didn't look closely enough and didn't see that.  Carry on!